_hooks/pre-commit

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+
+# A git hook script that checks for trailing whitespace
```

Comment:
>I recommend we keep the hooks in _hooks instead of hooks for the time being until it is possible to exclude dirs that have symlinks from the jekyll-watcher.

Yeah, this sounds good to me.